bodhi/client/bindings.py

```diff
@@ -592,7 +592,7 @@ def update_str(self, update, minimal=False):
             val += "\n  %s\n" % ('%supdates/%s' % (self.base_url,
                                                    update['alias']))
         else:
-            val += "\n  %s\n" % ('%s%s' % (self.base_url, update['title']))
+            val += "\n  %s\n" % ('%supdates/%s' % (self.base_url, update['title']))
```

Comment:
Oh good catch. I bet this ```else``` branch never happens in practice, but this was def wrong.